tensorflow/compiler/xla/service/gpu/llvm_gpu_backend/gpu_backend_lib.cc

```diff
@@ -86,14 +86,20 @@ static string GetSmName(std::pair<int, int> compute_capability) {
   int sm_version = 35;
   // If the current compute capability isn't known, fallback to the
   // most recent version before it.
-  for (int v : {75, 72, 70, 62, 61, 60, 53, 52, 50, 37, 35}) {
+  auto supported_version = {75, 72, 70, 62, 61, 60, 53, 52, 50, 37, 35};
```

Comment:
Does this create a dangling reference?  (I can never keep all of the C++ rules in my head.)

Reply:
That variable is inside a scope and doesn't leak to anywhere. So why do you think it could be a dangling reference?

Reply:
I was under the impression that the `{a, b, c}` syntax creates an `std::initializer_list` and that is only alive for the duration of the `{75, 72, 70, 62, 61, 60, 53, 52, 50, 37, 35};` statement.  Confusingly this seems to also have changed in C++14.  https://en.cppreference.com/w/cpp/utility/initializer_list

CC @timshen91 in case he has guidance here.